psl/psl/tests/common/asserts.rs

```diff
@@ -636,6 +665,23 @@ impl DefaultValueAssert for ast::Expression {
         self
     }
 
+    #[track_caller]
+    fn assert_uuid_version(&self, version: u8) -> &Self {
+        self.assert_uuid();
+
+        if let ast::Expression::Function(_, args, _) = self {
+            if let ast::Expression::NumericValue(actual, _) = &args.arguments[0].value {
+                assert_eq!(actual, &format!("{version}"));
```

Comment:
nitpicky, feel free to ignore:

```suggestion
                assert_eq!(actual, &version.to_string()));
```